### DISCUSSION TITLE
defining thumbnail for the video?

### DISCUSSION THREAD

**Discussion by eliramalho** [olive-editor/olive#2177] [Ideas]
Hi ! 
I've try to search the place to defining the thumbnail for the video (when it's stopping and awaiting to start) - but can't find it.
Normally that's it at export panel.
This is a great feature for marketing teams.

**Comment by mcDandy** [comment]
I have seen it done as additionall video track (works for VLC) or add
thumbnail as first frame which works with google photos.

Dne pá 24. 2. 2023 11:03 uživatel eliramalho ***@***.***>
napsal:
